src/core/xref-db.js

```diff
@@ -52,10 +53,22 @@ async function resolveFromCache(keys, cache) {
 }
```

Comment:
should reset time here. otherwise `VERSION_CHECK_WAIT` later, all `version` requests will be networked